docs/rpyc.rst

```diff
@@ -1,87 +1,114 @@
 distributed-computing. RPyC makes use of object-proxying, a technique that
 employs python’s dynamic nature, to overcome the physical boundaries between
 processes and computers, so that remote objects can be manipulated as if they
-were local. Here are simple steps you need to follow in order to install and
-use RPyC with ev3dev:
+were local.
 
-1. Install RPyC both on the EV3 and on your desktop PC. For the EV3, enter the
-   following command at the command prompt (after you `connect with SSH`_):
+For ev3dev, RPyC is most often used for:
+
+* robots that involve more than one EV3 (i.e. daisy chaining)
+* robots that perform some CPU intensive task (ex: Rubik's cube solver) where you
+  wish to run the CPU intensive part on your desktop PC
+
+Networking
+==========
+You will need to setup networking on your ev3dev device(s). The
+`ev3dev networking documentation <https://www.ev3dev.org/docs/networking/>`_ should get
+you up and running in terms of networking connectivity.
 
-   .. code-block:: shell
 
-       sudo easy_install3 rpyc
+Install
+=======
 
-   On the desktop PC, it really depends on your operating system. In case it is
-   some flavor of linux, you should be able to do
+1. RPyC is installed on ev3dev but we need to create a service that launches
+   ``rpyc_classic.py`` at bootup. Cut-n-paste the following commands on the
+   ev3dev device(s) that you wish to control via RPyC.
 
    .. code-block:: shell
 
-       sudo pip3 install rpyc
+      echo "[Unit]
+      Description=RPyC Classic Service
+      After=multi-user.target
 
-   In case it is Windows, there is a win32 installer on the project's
-   `sourceforge page`_. Also, have a look at the `Download and Install`_ page
-   on their site.
+      [Service]
+      Type=simple
+      ExecStart=/usr/bin/rpyc_classic.py
 
-2. Create file ``rpyc_server.sh`` with the following contents on the EV3:
+      [Install]
+      WantedBy=multi-user.target" > rpyc-classic.service
 
-   .. code-block:: shell
+      sudo cp rpyc-classic.service /lib/systemd/system/
+      sudo systemctl daemon-reload
+      sudo systemctl enable rpyc-classic.service
+      sudo systemctl start rpyc-classic.service
 
-      #!/bin/bash
-      python3 `which rpyc_classic.py`
 
-   and make the file executable:
+2. If you will be using an ev3dev device to control another ev3dev device you
+   can skip this step.  If you will be using your desktop PC to control an ev3dev
+   device you must install RPyC on your desktop PC. How you install RPyC depends
+   on your operating system. For Linux you should be able to do:
 
    .. code-block:: shell
 
-      chmod +x rpyc_server.sh
+       sudo apt-get install python3-rpyc
+
+   For Windows there is a win32 installer on the project's `sourceforge page`_.
+   Also, have a look at the `Download and Install`_ page on their site.
+
+Example
+=======
+The following python script should make a large motor connected to ``OUTPUT_A``
```

Comment:
I think more description is needed about which device this is run on and what it will do (operate on the remote device).